src/Uno.UI/UI/Xaml/Controls/SplitView/SplitView.mux.cs

```diff
@@ -0,0 +1,850 @@
+		}
+
+		FrameworkElement lightDismissLayerAsFE = GetTemplateChild<FrameworkElement>("LightDismissLayer");
+		if (lightDismissLayerAsFE is { })
+		{
+			// Uno TODO
+			// lightDismissLayerAsFE.Cast<FrameworkElement>()->put_AutomationPeerFactoryIndex(static_cast<INT>(KnownTypeIndex::SplitViewLightDismissAutomationPeer));
+
+			// Uno TODO
+			// lightDismissLayerAsFE.AllowsDragAndDropPassThrough = true;
+		}
+
+		FrameworkElement contentRootAsFE = GetTemplateChild<FrameworkElement>("ContentRoot");
+		if (contentRootAsFE is { })
+		{
+			m_tpContentRoot = contentRootAsFE;
+		}
+
+		COnApplyTemplate();
+	}
+
+	protected override void OnKeyDown(KeyRoutedEventArgs pArgs)
+	{
+		COnKeyDown(pArgs);
+		bool isHandled = pArgs.Handled;
+
+		if (!isHandled)
+		{
+			var displayMode = DisplayMode;
+			if (displayMode is SplitViewDisplayMode.CompactInline or SplitViewDisplayMode.CompactOverlay)
+			{
+				VirtualKey originalKey = pArgs.OriginalKey;
+
+				if (IsGamepadNavigationDirection(originalKey))
+				{
+					FocusNavigationDirection gamepadDirection = FocusSelection.GetNavigationDirection(originalKey);
+					SplitView spSplitView = this;
+
+					FocusManager pFocusManager = VisualTree.GetFocusManagerForElement(this)!;
+
+					XYFocusOptions xyFocusOptions = XYFocusOptions.Default;
+					xyFocusOptions.SearchRoot = spSplitView;
+					xyFocusOptions.ShouldConsiderXYFocusKeyboardNavigation = true;
+					xyFocusOptions.IgnoreClipping = false;
+					xyFocusOptions.IgnoreCone = true;
+
+					DependencyObject pCandidate = pFocusManager.FindNextFocus(new FindFocusOptions(gamepadDirection), xyFocusOptions);
+
+					if (pCandidate is { } && m_tpPaneRoot is { } && m_tpContentRoot is { })
+					{
+						// we got a candidate
+						bool shouldHandle = false;
+						bool contentHasFocus = false;
+						// IFC_RETURN(DXamlCore::GetCurrent()->GetPeer(pCandidate, &spCandidateDO));
+						DependencyObject spCandidateDO = pCandidate;
+
+						// if pane has focus and candidate is from content (or)
+						// content has focus and candidate is from the pane
+						// Then handle the key and move focus. If not, do not handle and let it bubble up to auto-focus
+						var paneHasFocus = m_tpPaneRoot.HasFocus();
+						if (paneHasFocus)
+						{
+							shouldHandle = m_tpContentRoot.IsAncestorOf(spCandidateDO);
+						}
+
+						if (!shouldHandle)
+						{
+							contentHasFocus = m_tpContentRoot.HasFocus();
+							if (contentHasFocus)
+							{
+								shouldHandle = m_tpPaneRoot.IsAncestorOf(spCandidateDO);
+							}
+						}
+
+						if (shouldHandle)
+						{
+							// Uno Specific: WinUI doesn't add the last argument (forceBringIntoView)
+							bool focusUpdated = this.SetFocusedElementWithDirection(spCandidateDO, FocusState.Keyboard, true, gamepadDirection, false);
+							pArgs.Handled = focusUpdated;
+						}
+					}
+				}
+			}
+		}
+	}
+
+	private void OnLoaded(object _, RoutedEventArgs __)
```

Comment:
This pattern is recognized by analyzers. See https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1801

```suggestion
	private void OnLoaded(object _1, RoutedEventArgs _2)
```